Server/DeviceManagement/DeviceManagement.Api.Http/DataMappers/UserDataMapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ModularHouse.Server.DeviceManagement.Domain.UserAggregate;
+using ModularHouse.Shared.Models.Responses.DMS;
+
+namespace ModularHouse.Server.DeviceManagement.Api.Http.DataMappers;
+
+public static class UserDataMapper
+{
+    public static CreatedUserResponse ToResponse(this User user)
+    {
+        return new CreatedUserResponse(user.Id, user.AdditionDate);
+    }
```

Comment:
2 options:
- Rename method to `ToCreatedResponse`;
- Rename response to `UserResponse`;

Both options are good in some way. Decide, what is better for you <3